styles.css

```diff
@@ -0,0 +1,138 @@
+
+body {
+    margin-left: 95px;
+    margin-right: 95px;
+    background-color: rgb(255, 250, 250);
+    max-width: 900px;
+    margin: auto auto;
+}
+
+h1 {
+    box-shadow: rgba(50, 50, 93, 0.25) 0px 30px 60px -12px inset, 
+    rgba(0, 0, 0, 0.3) 0px 18px 36px -18px inset;
+    text-align: center;
+    margin-top: 0;
+    margin-bottom: 10px;
+    font-size: 40px;
+    padding-top: 0;
+    padding-bottom: 0px;
+}
+
+.color, .font, .text {
+    display: block;
+    width: auto;
+    border:2px solid rgb(128, 128, 128);
+    margin: 10px auto 10px auto;
+    background-color: white;
+    box-shadow: rgba(50, 50, 93, 0.25) 0px 50px 100px -20px, 
+    rgba(0, 0, 0, 0.3) 0px 30px 60px -30px, 
+    rgba(10, 37, 64, 0.35) 0px -2px 6px 0px inset;
+}
+
+.color {
+    height: 370px;
+    margin-top: 0;
+}
+
+.font {
+    height: 450px;
+}
+
+.text {
+    height: 480px;
+
+}
+
+.content {
+    margin-left: 10px;
+    margin-right: 10px;
+}
+
+.title {
+    text-shadow: 2px 1px gray;
+    font-style: italic;
+    font-size: 28px;
+}
+
+.container {
+    display: flex;
+    margin: 0 10px 0 10px;
+}
+
+.column {
+    flex: 25%;
+    height: 200px;
+    padding: 10px;
+    margin: 5px;
+    text-align: center;
+    border:2px solid black;
+    border-radius: 1em;
+}
+
+#sans {
+    margin: 30px 0 20px 20px;
+}
+
+#sans1 {
+    font-family: SansReg;
+}
+
+#sans2 {
+    font-family: SansBold;
+}
+
+#sans3 {
+    font-family: SansItalic;
+}
+
+#sans h2 {
+    font-family: SansReg;
+    text-decoration: underline;
+    font-weight: bold;
+}
+
+#spectral {
+    margin: 30px 0 20px 20px;
+}
+
+#spec1 {
+    font-family: SpectralReg;
+}
+#spec2 {
+    font-family: SpectralBold;
+}
+#spec3 {
+    font-family: SpectralItalic;
+}
+#spectral h2 {
+    font-family: SpectralReg;
+    text-decoration: underline;
+    font-weight: bold;
+}
+
+.h1 {
+    font-family: SansSemiBold;
+    font-size: 46px;
+    margin-top: 5px;
+    margin-bottom: 5px;
+}
+
+.h2 {
+    font-family: SpectralMedium;
```

Comment:
any reason for different fonts for h1 and h2? I would've expected one font for headers, and the second font for regular text. But that's just opinion